apps/web/src/pages/_app.tsx

```diff
@@ -13,6 +13,7 @@ import { OpenAPI } from '@web/client';
 export default function App({ Component, pageProps, ...appProps }: AppProps) {
   OpenAPI.CREDENTIALS = 'include';
   OpenAPI.WITH_CREDENTIALS = true;
+  OpenAPI.BASE = process.env.BACKEND_DOMAIN || 'https://localhost:8080';
```

Comment:
we probably don't need this if we use the reverse proxy, since they'll be seen as hosted on the same domain

Reply:
on my end it doesn't work unless I add this, not sure if this is a problem on my machine?